odoo/model/invoice_category.go

```diff
@@ -0,0 +1,86 @@
+package model
+
+import (
+	"context"
+
+	"github.com/vshn/appuio-odoo-adapter/odoo"
+)
+
+// InvoiceCategory (alias "Section" in Invoices) visually categorizes line items into logical groups.
+type InvoiceCategory struct {
+	// ID is the data record identifier.
+	ID int `json:"id,omitempty"`
+	// Name is the title of the category/section within an invoice.
+	Name string `json:"name,omitempty"`
+	// Sequence controls how line items are grouped together and how each category is ordered within an invoice.
+	//
+	// If there are line items that share the same category, but each category has a unique sequence number, then all line items are grouped together.
+	// In this case the order of line items in the invoice doesn't matter.
+	// The ordering among the categories themselves is ascending with the category with the lowest sequence number appearing at the top.
+	//
+	// If there are categories that share the same sequence number then the ordering of line items greatly matters within an invoice.
+	// Line items that share the same category but aren't sequentially defined in the invoice end up distributed ("as is") with the category appearing multiple times.
+	Sequence int `json:"sequence,omitempty"`
```

Comment:
I'll create a separate documentation PR that provides examples with images. I think it's a pretty important detail that we need to manage.

Reply:
See #5